examples/src/components/App.jsx

```diff
@@ -27,11 +27,13 @@ const App = () => (
     </div>
 
     <p className={styles.description}>
-      {"hoc-react-loader .... FIXME"}
+      {"hoc-react-loader is an higher order component to call a function whenever "}
```

Comment:
... calling a function when a watched prop is not defined (unless null does'nt show the loader) ....
... (It shows a loading component) when it's waiting for the props to be defined. ....